Formula/go.rb

```diff
@@ -60,6 +60,9 @@ def install
     # Remove useless files.
     # Breaks patchelf because folder contains weird debug/test files
     rm_rf Dir[libexec/"src/debug/elf/testdata"]
+
+    # Remove 32-bit test data
+    rm_rf Dir[libexec/"src/runtime/pprof/testdata/test32"]
```

Comment:
```suggestion
    (libexec/"src/runtime/pprof/testdata/test32").rmtree
```
From https://github.com/Homebrew/homebrew-core/pull/82830#discussion_r684678216